src/utils/format.ts

```diff
@@ -0,0 +1,25 @@
+export const dateFormat = (targetDate: Date) => {
```

Comment:
<img width="380" alt="image" src="https://github.com/woowacourse/react-shopping-cart/assets/60066472/99058815-b664-4425-bc20-b392ea7c935f">


유틸함수의 반환값을 주석만 보고 빠르게 예상할 수 있도록 작성해도 좋겠습니다.

```suggestion
/** @returns '2024년 5월 1일' 꼴로 포맷 */
export const dateFormat = (targetDate: Date) => {
```

Reply:
> 유틸함수의 반환값을 주석만 보고 빠르게 예상할 수 있도록 작성해도 좋겠습니다.

이럴 때 활용해주어도 직관적으로 파악하는데 많은 도움이될 것 같네요!! 감사합니다 :)